nj2k/src/org/jetbrains/kotlin/nj2k/tree/statements.kt

```diff
@@ -88,6 +88,8 @@ class JKKtConvertedFromForLoopSyntheticWhileStatement(
 ) : JKStatement() {
     var variableDeclaration: JKStatement by child(variableDeclaration)
     var whileStatement: JKWhileStatement by child(whileStatement)
+    // For the use in ContinueStatementConverter
+    var forLoopUpdaters: List<JKStatement> = listOf()
```

Comment:
For storing `JKElement`, please use the `by child/children` approach. It allows to control that we will not use the same element a child twice